neo/conf/application.conf

```diff
@@ -76,6 +76,7 @@ bounded-mailbox {
 }
 
 neo.timeout.initial = "10 minutes"
+neo.stream.max.frame.length = "8388608 Bytes"
```

Comment:
I think let's just stick with this number for now. 
in terms of "good code", it's better to enable something like
`neo.stream.max.frame.length = "8MB"`
or simply
`neo.stream.max.frame.megabytes = 8`
to make more sense 😁